pkg/command/context.go

```diff
@@ -99,9 +102,14 @@ func init() {
 
 	initCreateCtxCmd()
 
-	listCtxCmd.Flags().StringVarP(&targetStr, "target", "t", "", "list only contexts associated with the specified target (kubernetes[k8s]/mission-control[tmc]/application-engine[tae])")
+	listCtxCmd.Flags().StringVarP(&targetStr, "target", "", "", "list only contexts associated with the specified target (kubernetes[k8s]/mission-control[tmc]/application-engine[tae])")
```

Comment:
As discussed offline, we agreed we do not want to provide TAE as a valid value for the `--target` flag in context commands. If the user wants to use the TAE context-type, they should use the `--type/-t` flag instead of the deprecated `--target` flag.

Reply:
I have updated the PR accordingly.